modules/sandboxed-containers-installing-operator-cli.adoc

```diff
@@ -3,5 +3,107 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-installing-operator-cli_{context}"]
+= Installing the {sandboxed-containers-operator} using the CLI
 
-= Installing the Sandboxed Containers Operator using the CLI
+You can install the {sandboxed-containers-operator} using the {product-title} CLI.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have subscribed to the {sandboxed-containers-first} catalog.
++
+[NOTE]
+====
+Subscribing to the {sandboxed-containers-first} catalog provides `openshift-sandboxed-containers` namespace access to the {sandboxed-containers-operator}.
+====
+
+.Procedure
+
+. Create a YAML file that contains the following manifest:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-sandboxed-containers-operator
+
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-sandboxed-containers-kataconfig-group
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  targetNamespaces:
+    - openshift-sandboxed-containers
+
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sandboxed-containers-operatorhub
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  name: sandboxed-containers-kataconfig
+  startingCSV: openshift-sandboxed-containers-kataconfig.v1.0.0
+  channel: "preview-1.0"
+  approval: "Automatic"
+----
++
+[NOTE]
+====
+Using the *preview-1.0* channel ensures that you install the version of {sandboxed-containers-first} that is compatible with your {product-title} version.
+====
+
+. Create the required `Namespace`, `OperatorGroup`, and `Subscription` objects for {sandboxed-containers-first}:
++
+[source,terminal]
+----
+$ oc apply -f <file name>.yaml
+----
```

Comment:
Do we have any rules about `oc apply` vs. `oc create`? I am under the impression that `create` is for making new objects and apply is for modifying existing objects.